src/main/java/berlin/yuna/nano/services/metric/logic/MetricService.java

```diff
@@ -28,6 +28,10 @@
 @SuppressWarnings({"unused", "UnusedReturnValue"})
 public class MetricService extends Service {
     private final MetricCache metrics = new MetricCache();
+    private String prometheusPath;
```

Comment:
no `private`, use protected. We are not hiding something from the developers. When someone wants to override something, they are free to do it